test/thermometer.js

```diff
@@ -377,6 +377,169 @@ exports["Thermometer -- ANALOG"] = {
   },
 };
 
+function createMAX31850K(pin, address) {
+  return new Thermometer({
+    controller: "MAX31850K",
+    pin: pin,
+    address: address,
+    freq: 100,
+    board: this.board
+  });
+}
+
+exports["Thermometer -- MAX31850K"] = {
+
+  setUp: function(done) {
+    this.pin = 2;
+    this.sendOneWireConfig = this.sandbox.spy(MockFirmata.prototype, "sendOneWireConfig");
+    this.sendOneWireSearch = this.sandbox.spy(MockFirmata.prototype, "sendOneWireSearch");
+    this.sendOneWireDelay = this.sandbox.spy(MockFirmata.prototype, "sendOneWireDelay");
+    this.sendOneWireReset = this.sandbox.spy(MockFirmata.prototype, "sendOneWireReset");
+    this.sendOneWireWrite = this.sandbox.spy(MockFirmata.prototype, "sendOneWireWrite");
+    this.sendOneWireWriteAndRead = this.sandbox.spy(MockFirmata.prototype, "sendOneWireWriteAndRead");
+
+    done();
+  },
+
+  tearDown: function(done) {
```

Comment:
Just kidding, I forgot this file had been renovated to include a global tearDown